src/main/resources/features/moth/55 - Print MOT certificate successfully.feature

```diff
@@ -0,0 +1,41 @@
+@mothprint @mothint
+Feature: 54 - Check whether MOT certificates can be downloaded after entering the V5C number
```

Comment:
Feature number should be changed from 54 to 55 to match the file

Reply:
Number changed.